src/main/java/se/kth/bbc/project/ProjectsManagementBean.java

```diff
@@ -61,8 +67,14 @@ public void setAllProjects(List<ProjectsManagement> allProjects) {
     return allProjects;
   }
 
-  public int getHdfsQuota() {
-    return projectsManagementController.getHdfsQuota();
+  public int getHdfsQuota(String projectname) throws IOException {
+    long quota = projectsManagementController.getHdfsQuota(projectname);
+    this.hdfsquota = String.valueOf(quota);
+    return (int) quota;
+  }
```

Comment:
I can add it but it handles it by itself since it knows it is looking for a number
![selection_026](https://cloud.githubusercontent.com/assets/7311243/11637084/eee5dd36-9d1f-11e5-859c-0e72c2ca99e6.jpg)
you are not able to submit the change and there is also a growl message saying why